content/bindings/jmeos.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "JMEOS"
+date: 2024-08-31T12:13:56+02:00
+draft: false
+---
+
+JMEOS: A Java binding of the MEOS spatiotemporal library
+
+Short description: TODO
+
+Link to API docs: https://1-arijits.github.io/JMEOS/
```

Comment:
I know the link will only work after the actions run for the first time, but we should probably already put the link to https://mobilitydb.github.io/JMEOS/ here

Reply:
okay i will do that

Reply:
i have changed the link now you can merge